docs/en/upload.md

```diff
@@ -62,11 +62,12 @@ print("Connecting to WiFi access point...")
 wifi.setmode(wifi.STATION)
 wifi.sta.config(SSID, PASSWORD)
 -- wifi.sta.connect() not necessary because config() uses auto-connect=true by default
-tmr.alarm(1, 1000, 1, function()
+wifitimer = tmr.create()
+wifitimer:alarm(1000, tmr.ALARM_AUTO, function(cb_timer)
     if wifi.sta.getip() == nil then
         print("Waiting for IP address...")
     else
-        tmr.stop(1)
+        cb_timer:unregister()
```

Comment:
True, thanks for the reminder, will fix. The reference to `wifitimer` is no longer necessary.